api-audit/src/main/java/com/capitalone/dashboard/evaluator/Evaluator.java

```diff
@@ -38,9 +38,10 @@
 
     public  abstract Collection<T> evaluate(Dashboard dashboard, long beginDate, long endDate, Map<?, ?> data) throws AuditException;
 
-
     public abstract T evaluate(CollectorItem collectorItem, long beginDate, long endDate, Map<?, ?> data) throws AuditException, HygieiaException;
 
+    public abstract T evaluate(CollectorItem collectorItem, List<CollectorItem> collectorItemList, long beginDate, long endDate, Map<?, ?> data) throws AuditException, HygieiaException;
```

Comment:
@subodhbattina  This method is only used in CodeReviewEvaluator, This can go in as private method in specific evaluator instead of base class.

Reply:
@nireeshT I see what you are saying, I can remove the overloaded evaluate method from the parent Evaluator class. But was wondering if there will be a need for this overloaded method in the future for any of the existing or any new evaluator classes in the future.

Reply:
@nireeshT Have converted the method to a local method, thanks.